.github/workflows/release.yml

```diff
@@ -44,33 +36,17 @@ jobs:
           cd dist/packages/ngx-web-component && npm publish --access=public
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - name: Determine tag
-        id: determine_tag
-        run: |
-          echo "TAG_VERSION=$(npm view node_modules/@readalongs/web-component version)" >> $GITHUB_OUTPUT
-      - name: Bump version and push tag
-        id: tag_version
-        uses: mathieudutour/github-tag-action@v6.1
+      - name: Update CHANGELOG
+        id: changelog
+        uses: requarks/changelog-action@v1
         with:
```

Comment:
it might be nice to have a check here that determines whether npm has the version available (i.e. whether we forgot to bump). `npm view @readalongs/web-component version` should display the most recent version, and that should be less than the current defined version in the code/tag

Reply:
if it doesn't, we shouldn't create a github release, and this should fail.

Reply:
Will `npm publish` fail if you try to republish the same version again?

Reply:
But good point, we should make sure the GH release doesn't happen in this case.

Reply:
Indeed, we should already be safe: https://docs.npmjs.com/cli/v8/commands/npm-publish says:
> The publish will fail if the package name and version combination already exists in the specified registry.
and once the first publish fails the entire workflow will abort.

Reply:
hmm I feel like if that's the case, it's new. I seem to remember things not aborting just because npm publish failed